changelog.md

```diff
@@ -10,6 +10,14 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.7.1 March 17, 2021
+* Fixed the `help` dialog incorrectly linked to the changelog of TinyMCE 4 instead of TinyMCE 5.
```

Comment:
```suggestion
* Fixed the `help` dialog incorrectly linking to the changelog of TinyMCE 4 instead of TinyMCE 5.
```

We'll also fix this in TinyMCE tomorrow